# MIT License
#
# # Copyright (c) 2022 Saurabh Gupta, Ignacio Vizzo, Cyrill Stachniss, University of Bonn
#
# Permission is hereby granted, free of charge, to any person obtaining a copy
# of this software and associated documentation files (the "Software"), to deal
# in the Software without restriction, including without limitation the rights
# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
# copies of the Software, and to permit persons to whom the Software is
# furnished to do so, subject to the following conditions:
#
# The above copyright notice and this permission notice shall be included in all
# copies or substantial portions of the Software.
#
# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
# SOFTWARE.

# VDBFUSION
voxel_size: 0.02
sdf_trunc: 0.06
space_carving: True

# Triangle Mesh Generation
fill_holes: True
min_weight: 10.0

# PointCloud
apply_pose: True
preprocess: True
min_range: 0.1
max_range: 5.0
pcl_topic: "/camera/depth_registered/points"

# Transform
use_tf_transforms: False
tf_topic: "/kinect/vrpn_client/estimated_transform"
timestamp_tolerance_ns: 1000000

# Static Transform
invert_static_tf: True
# Translation
tx: 0.00114049
ty: 0.0450936
tz: 0.0430765
# Rotation Quaternion
x: 0.0924132
y: 0.0976455
z: 0.0702949
w: 0.9884249
